.github/workflows/cypress.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         node-version: [20]
         php-versions: ['8.1']
         databases: ['sqlite']
-        server-versions: ['stable28', 'master']
+        server-versions: ['stable28', 'stable29', 'stable30', 'master']
```

Comment:
To be honest I think it's a bit waste of CI minutes to run Cypress against each supported server branch. Running it against the lowest supported and master so far worked well for me in other apps. What do you think @nickvergessen?

Reply:
I think we end up here every time :D 
Fine by me, just adjust it and merge

Reply:
We don't have to change anything then :tada: